src/Round.js

```diff
@@ -4,23 +4,41 @@ import Puzzle from './Puzzle.js';
 import Wheel from './Wheel.js';
 
 class Round {
-  constructor(player, puzzle, game, wheel) {
+  constructor(players, playerOne, playerTwo, playerThree, currentPlayer, puzzle, game, wheel) {
     this.players = game.players;
     this.puzzle = this.getNewPuzzle()
-    this.wheel = wheel
-  
+    this.wheel = wheel;
+    this.player1 = new Player(playerOne);
+    this.player2 = new Player(playerTwo);
+    this.player3 = new Player(playerThree);
+    this.currentPlayer = currentPlayer; 
+    // this.currentTurn = this.startTurn();
+  }
 
+  getStartingPlayer() {
```

Comment:
At the start of the game we can assign the first player to always be ```newGame.players[0]```, so we don't necessarily need to find the first player. Then when we need to find the next player during the game we will invoke the switchPlayer method below, which looks great. Let me know what you think about deleting the getStartingPlayer method.

Reply:
Agree about not needing the getStartingPlayer method, unless we want to use that for rounds after round 1, in which case we can add in conditional logic about who has the most/least money.